app/controllers/claims/pages_controller.rb

```diff
@@ -1,2 +1,3 @@
 class Claims::PagesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Should we just put this in the ApplicationController? And if any controllers don't need it we can skip it in there.

Reply:
I agree with Catalin that this would be better on the ApplicationController and we skip it when not needed. It is easier to require auth by default since that is what we will need most often.

Reply:
I think I'd propose something slightly different.

```
class Claims::PagesController < Claims::Base::ApplicationController # Base / Users / Main any of these namespaces work for me
end

class Claims::Base::ApplicationController < ApplicationController
  before_action :authenticate_user!
end
```

This way, we have a base controller for general user routes and one for support. We also have the top-level `ApplicationController`'s action hooks unmodified and likely the path of least surprise. Anything inheriting from `ApplicationController` should have access to all helpers and custom controller methods, but without any additional behaviour added.

What do you both think?

Reply:
I think auth by default is also viable now that I think about it though 🤔 .

Reply:
Yeah, scratch the above suggestion. Keep it simple.

Reply:
I guess I don't really see the value of having ` Claims::Base::ApplicationController` at the moment, feels like we are abstracting too soon.

I like the idea of auth by default and if you need to skip it for the support controller you can skip it or override the method from the `ApplicationController`.

In the end, I don't really have a strong opinion on this. Happy with both approaches